### PR TITLE
Changed rainy observation check 

### PR DESCRIPTION
**Pull request by mvyboh**
- changed the way rainy observations are checked so that all radars from last timestamp are checked (before user selected only 1)
- added support of rainbow files compressed to tar ball
- possible fix of memory leak issue of pyart module functions
- minor code cleaning and fixing

### CODE REVIEW

File: rainbow_to_h5.py [hunk 1]
```diff
@@ -104,68 +99,69 @@ def convert_rnbw_to_h5(conf, all_files, outfiles, datetimes_str, rain_thres):
...
-                # to np.array from Grid object
-                data = grid.fields[RAINBOW_FIELD_NAMES[product]]["data"][0]
+                    # to np.array from Grid object
+                    data = grid.fields[RAINBOW_FIELD_NAMES[product]]["data"][0]
+
+                    # remove clutter from image
+                    if product in ['dBZ', 'dBZv']:
+                        clmap = classify.filter_gabella(data, rm_nans=False, cartesian=True)
+                        data[np.nonzero(clmap)] = np.nan
+                        clmap = classify.filter_gabella(data, rm_nans=False, cartesian=True)
```
Comment (pavlikp) [inline rainbow_to_h5.py]: Toto je tu naschval 2-krat?

Comment (pavlikp) [reply rainbow_to_h5.py]: to iste aj riadok 124

Comment (mvyboh) [reply rainbow_to_h5.py]: fixnem v dalsom commite

Comment (mvyboh) [reply rainbow_to_h5.py]: zmenene v https://github.com/kinit-sk/radar-dataset-preprocess/pull/1/commits/c27d9122555137ed3ce0fa99bedc5591e9ccae83

**Pull request merged**